room.go

```diff
@@ -49,6 +64,9 @@ func (r *Room) Draw() {
 	for _, item := range items {
 		item.Draw()
 	}
+
+	// TODO if Debug Mode
+	r.wbmatrix.Draw()
```

Comment:
I don't like this TODO and the boxes being rendered by default. I'd rather remove the line and implement walkbox debugging mode using a key and managed by the user control.

Reply:
Yep, this is the idea, connecting what is done in https://github.com/apoloval/pctk/pull/37

Let me remove it and add it when debug be implemented